docs/source/tensors.rst

```diff
@@ -8,7 +8,7 @@ torch.Tensor
 A :class:`torch.Tensor` is a multi-dimensional matrix containing elements of
```

Comment:
Please undo the changes to this file. They are not part of task.

Reply:
no worries, I've reverted this file when importing to Phabricator (I need change an internal BUCK reference to the renamed file, too).